examples/js/loaders/MTLLoader.js

```diff
@@ -396,6 +396,18 @@ THREE.MTLLoader.MaterialCreator.prototype = {
 
 					break;
 
+				case 'map_ks':
+
+					// Specular map
+
+					if ( params.specularMap ) break; // Keep the first encountered texture
+					console.log("here");
```

Comment:
daaaaang it. I removed it in my pull request against master. Can't catch a break.
So, Do I redact my PR and do it again or something else?